tests/test_openclhpp.cpp

```diff
@@ -3833,4 +3880,103 @@ void testSemaphoreRetain(void) {}
 void testSemaphoreRelease(void) {}
 #endif // cl_khr_semaphore
 
+/****************************************************************************
+ * Tests for cl_khr_external_memory
+ ****************************************************************************/
+static cl_int clEnqueueAcquireExternalMemObjectsKHR_testEnqueueAcquireExternalMemObjects(
```

Comment:
I think these tests are all OK as-is, but if you want to be on the safe side you could consider wrapping them all inside of one big ifdef for `cl_khr_external_memory` similar to what was done for the `cl_khr_semaphore` tests above.

Reply:
Done.